pkgdb2/ui/packages.py

```diff
@@ -1232,6 +1232,15 @@ def package_request_new():
         pkg_poc = flask.g.fas_user.username
         pkg_upstream_url = form.upstream_url.data
 
+        bz = APP.config.get('PKGDB2_BUGZILLA_URL')
+        if bz not in pkg_review_url:
+            try:
+                int(pkg_review_url)
```

Comment:
What would happen if bugzilla_url is https://bugzilla.redhat.com/, but I enter http://bugzilla.redhat.com/123456?
The URL would become http://.../123456/https://....
(or if I made a typo in the URL).

(just a note, I think people will just copy-paste normally)